helper-cli/src/main/kotlin/commands/repoconfig/GenerateScopeExcludesCommand.kt

```diff
@@ -145,6 +145,16 @@ private fun getScopeExcludesForPackageManager(packageManagerName: String): List<
                 reason = ScopeExcludeReason.DEV_DEPENDENCY_OF,
                 comment = "Packages for static code analysis only."
             ),
+            ScopeExclude(
+                pattern = ".*debug.*",
+                reason = ScopeExcludeReason.DEV_DEPENDENCY_OF,
+                comment = "Packages for debug builds."
+            ),
+            ScopeExclude(
+                pattern = ".*Debug.*",
```

Comment:
As these are regexes, do you think it makes sense to combine this (and the previous similarly) to `".*[Dd]ebug.*"`?

Reply:
I believe it's better to not combine them, because it seems that it most of the cases leads to simpler entries in the `ort.yml`.